airflow/providers/amazon/aws/hooks/ec2.py

```diff
@@ -33,21 +33,90 @@ class EC2Hook(AwsBaseHook):
         :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
     """
 
+    # Describe response
+    RESERVATIONS = 'Reservations'
+    INSTANCES = 'Instances'
+    STATE = 'State'
+    NAME = 'Name'
+    INSTANCE_ID = 'InstanceId'
+
     def __init__(self,
                  *args,
                  **kwargs):
-        super().__init__(resource_type="ec2", *args, **kwargs)
+        super().__init__(client_type="ec2", *args, **kwargs)
 
-    def get_instance(self, instance_id: str):
```

Comment:
This is only acceptible since the Ec2Hook hasn't yet been included in a release -- otherwise this would be a breaking change and you would have have to include a compatibility layer.

Reply:
missed this, please let me know if this needs to be taken care off or we are good for now.

Reply:
_Unfortunately_ the ec2 integration was released. So we would need take care of it like ash mentioned.

Reply:
👍